python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -57,19 +58,39 @@ class AssembleCoaddConfig(CoaddBaseTask.ConfigClass):
         length=2,
         default=(2000, 2000),
     )
+    statistic = pexConfig.ChoiceField(
+        dtype=str,
+        doc="Main stacking statistic for aggregating over the epochs.",
+        default="MEANCLIP",
+        allowed={
+            "MEAN": "unclipped mean",
+            "MEANCLIP": "N-sigma clipped mean (N set by sigmaClip)",
+            "MEDIAN": "median",
+            "NPOINT": "number of epochs",
+            "STDEV": "standard deviation",
+            "VARIANCE": "variance",
+            "IQRANGE": "inter-quartile range",
+            "STDEVCLIP": "N-sigma clipped stdev (N set by sigmaClip)",
+            "VARIANCECLIP": "N-sigma clipped variance (N set by sigmaClip)",
+            "MIN": "minimum",
+            "MAX": "maximum",
+            "SUM": "sum",
+            "MEANSQUARE": "mean of square of  pixel values",
+        }
+    )
```

Comment:
Why do we need to specify these?  There's code in Statistics that takes a string and converts it to a valid enum;  why don't we use that in the validate method?

Listing statistics here is what caused this problem in the first place.